src/main/webapp/js/instructorSearch.js

```diff
@@ -8,3 +8,11 @@ $(function() {
 	
 	$("div[id^=plainCommentText]").css("margin-left","15px");
 });
+
```

Comment:
Hi @kanghj: Is there a way to do it directly in github? could you please guide me how to do it. because I haven't added anything in this file, tried to remove before pushing but couldn't.